mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/routes/v1/VirksomhetRoutes.kt

```diff
@@ -35,6 +44,33 @@ fun Route.virksomhetRoutes() {
             }
         }
 
+        get("{orgnr}/kontaktperson") {
+            val orgnr = call.parameters.getOrFail("orgnr")
+
+            if (!OrgnummerUtil.erOrgnr(orgnr)) {
+                throw BadRequestException("Verdi sendt inn er ikke et organisasjonsnummer. Organisasjonsnummer er 9 siffer og bare tall.")
+            }
+
+            call.respond(virksomhetService.hentKontaktpersoner(orgnr))
+        }
```

Comment:
Hvordan blir dette når vi på en gjennomføring kan ha en eller flere kontaktpersoner for en virksomhet. Burde vi skille på om man henter kontaktperson fra avtale eller fra gjennomføring med en gang?

Reply:
Kan en gjennomføring ha flere på én virksomhet?

Reply:
I følge denne oppgaven https://trello.com/c/Ju5h2KKW/908-gjennomf%C3%B8ringskjemaet-legg-til-kontaktperson-hos-arrang%C3%B8ren kan det det, men kanskje vi skal høre med Marthe (og eller betabrukerne) hva de tenker om det.

Reply:
Ja ok. Men vet ikke om det har noe å si på dette endepunktet? Her bare henter man alle til en virksomhet uavhengig av avtaler og gjennnomføringer. Koblingen på gjennomføring blir vel en join tabell i tillegg.

Reply:
Skjønner, da er det sikkert greit